### PR TITLE
[Ryujinx.Cpu] Address dotnet-format issues

### PR DESCRIPTION
**Pull request by TSRBerry**
Base PR: #4670

### CODE REVIEW

File: src/Ryujinx.Cpu/AddressSpace.cs [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
+using Ryujinx.Common.Utilities;
 using Ryujinx.Memory;
 using System;
 
 namespace Ryujinx.Cpu
 {
     public class AddressSpace : IDisposable
     {
+#pragma warning disable IDE0051 // Remove unused private member
         private const ulong PageSize = 0x1000;
```
Comment (AcK77) [inline src/Ryujinx.Cpu/AddressSpace.cs:11]: If it's not used, maybe we should remove it

File: src/Ryujinx.Cpu/AddressSpace.cs [hunk 2]
```diff
@@ -391,7 +393,9 @@ private void MapPrivate(ulong va, ulong size)
             ulong vaAligned = BitUtils.AlignDown(va, alignment);
             ulong endAddressAligned = BitUtils.AlignUp(endAddress, alignment);
 
+#pragma warning disable IDE0059 // Remove unnecessary value assignment
             ulong sizeAligned = endAddressAligned - vaAligned;
```
Comment (AcK77) [inline src/Ryujinx.Cpu/AddressSpace.cs:397]: Maybe we should do something with it, or remove the assignment.

File: src/Ryujinx.Cpu/AddressSpace.cs [hunk 3]
```diff
@@ -436,7 +440,9 @@ private void UnmapPrivate(ulong va, ulong size)
                 return;
             }
 
+#pragma warning disable IDE0059 // Remove unnecessary value assignment
             ulong alignedSize = endAddressAligned - vaAligned;
```
Comment (AcK77) [inline src/Ryujinx.Cpu/AddressSpace.cs:444]: Maybe we should do something with it, or remove the assignment.

File: src/Ryujinx.Cpu/AppleHv/HvApi.cs [hunk 4]
```diff
@@ -1,12 +1,12 @@
-using ARMeilleure.State;
 using System;
 using System.Runtime.InteropServices;
 
 namespace Ryujinx.Cpu.AppleHv
 {
+#pragma warning disable IDE1006 // Naming rule violation
```
Comment (AcK77) [inline src/Ryujinx.Cpu/AppleHv/HvApi.cs:6]: Maybe we should rename the structs then

File: src/Ryujinx.Cpu/MemoryEhMeilleure.cs [hunk 5]
```diff
@@ -10,20 +10,17 @@ public class MemoryEhMeilleure : IDisposable
...
 
         public MemoryEhMeilleure(MemoryBlock addressSpace, MemoryBlock addressSpaceMirror, MemoryTracking tracking)
         {
-            _tracking = tracking;
-
             _baseAddress = (ulong)addressSpace.Pointer;
             ulong endAddress = _baseAddress + addressSpace.Size;
 
-            _trackingEvent = new TrackingEventDelegate(tracking.VirtualMemoryEvent);
+            _trackingEvent = tracking.VirtualMemoryEvent;
```
Comment (AcK77) [inline src/Ryujinx.Cpu/MemoryEhMeilleure.cs:23]: Strange change, is `VirtualMemoryEvent` already a delegate ?

File: src/Ryujinx.Cpu/MemoryHelper.cs [hunk 6]
```diff
@@ -42,22 +41,20 @@ public static void FillWithZeros(IVirtualMemoryManager memory, ulong position, i
 
         public static string ReadAsciiString(IVirtualMemoryManager memory, ulong position, long maxSize = -1)
         {
-            using (RecyclableMemoryStream ms = MemoryStreamManager.Shared.GetStream())
+            using RecyclableMemoryStream ms = MemoryStreamManager.Shared.GetStream();
```
Comment (AcK77) [inline src/Ryujinx.Cpu/MemoryHelper.cs:44]:
```suggestion
            using RecyclableMemoryStream ms = MemoryStreamManager.Shared.GetStream();

```

**Review by AcK77 — CHANGES_REQUESTED** [review]
Could be nice to get the namespace issue fixed too.

**Comment by TSRBerry** [comment]
I probably need to address namespaces with #4670 anyway

File: src/Ryujinx.Cpu/MemoryEhMeilleure.cs (hunk 5 above)
Comment (TSRBerry) [reply src/Ryujinx.Cpu/MemoryEhMeilleure.cs:23]: Yes!

**Comment by TSRBerry** [comment]
Checks are passing here as well, marking as ready!

**Review by marysaka — APPROVED** [review]

File: src/Ryujinx.Cpu/AddressSpace.cs (hunk 2 above)
Comment (AcK77) [reply src/Ryujinx.Cpu/AddressSpace.cs:393]: Still unresolved

File: src/Ryujinx.Cpu/AddressSpace.cs (hunk 3 above)
Comment (AcK77) [reply src/Ryujinx.Cpu/AddressSpace.cs:440]: Still unresolved

File: src/Ryujinx.Cpu/AppleHv/HvApi.cs [hunk 7]
```diff
@@ -261,60 +260,60 @@ static partial class HvApi
         public const string LibraryName = "/System/Library/Frameworks/Hypervisor.framework/Hypervisor";
 
         [LibraryImport(LibraryName, SetLastError = true)]
```
Comment (AcK77) [inline src/Ryujinx.Cpu/AppleHv/HvApi.cs:262]: Maybe we could use `EntryPoint` args to map the real name `EntryPoint = "hv_vm_get_max_vcpu_count"` and then use a consistent name like `GetMaxVCPUCount`.
Same comment is available for other `LibraryImport`.

File: src/Ryujinx.Cpu/AddressSpace.cs [hunk 8]
```diff
@@ -495,9 +497,11 @@ private static bool CanCoalesce(PrivateMapping left, PrivateMapping right)
 
         public void Dispose()
         {
+            GC.SuppressFinalize(this);
```
[the remark this answers is not in the document]
Comment (TSRBerry) [reply src/Ryujinx.Cpu/AddressSpace.cs:500]: Do we do that in a separate PR later like with the other PR?

[the remark this answers is not in the document]
Comment (gdkchan) [reply src/Ryujinx.Cpu/AddressSpace.cs:500]: Yea.

**Review by gdkchan — APPROVED** [review]
lgtm, thanks.

**Review by AcK77 — APPROVED** [review]

**Pull request merged**